tests/plugins/test_pin_operator_digests.py

```diff
@@ -128,7 +128,7 @@ def mock_digest_query(pullspec, digest):
     url = 'https://{}/v2/{}/{}/manifests/{}'.format(i.registry, i.namespace, i.repo, i.tag)
     headers = {
         'Content-Type': MEDIA_TYPE_DOCKER_V2_MANIFEST_LIST,
-        'Docker-Content-Digest': digest
```

Comment:
IMHO `digest` var itself should contain that prefix

Reply:
ok